packaging_automation/citus_package.py

```diff
@@ -31,14 +31,19 @@
 POSTGRES_EXCLUDE_FILE_NAME = "pg_exclude.yml"
 
 docker_image_names = {
+    "almalinux": "almalinux",
+    "rockylinux": "almalinux",
     "debian": "debian",
+    "el/9": "almalinux-9",
```

Comment:
Before this change, I was only using OS name to define docker image name. Sincei both centos and almalinux is being used as base image now, I added os version as well